src/obsidian_ai_hub/agents/registry.py

```
@@ -0,0 +1,287 @@
+    for tool_id, meta in TOOL_DEFINITIONS.items():
+        catalog.append(
+            {
+                "tool_id": tool_id,
+                "name": meta["name"],
+                "description": meta["description"],
+            }
+        )
+    return catalog
+
+
+def resolve_tools(tool_ids: Sequence[str]) -> List[BaseTool]:
```

Ruff (0.16.1)

[error] Undefined name `Sequence`

(F821)